tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+  "include": ["src", "types"],
```

We don't have a `types` directory.
```suggestion
  "include": ["src"],
```